manifest.json

```diff
@@ -1463,6 +1463,13 @@
      "description": "Offers improvements such as highlighted comments and autocomplete for brackets, quotes and more. ([demo](https://s12.gifyu.com/images/Sckre.gif))",
      "remote": "https://github.com/chqs-git/code-plus.git:b336999b707709df2ddcb865f664623c5beceabb",
      "mod_version": 3
+    },
+    {
+      "description": "Theme manager with base16 color schemes for lite-xl",
+      "version": "0.1",
+      "remote": "https://github.com/SmileYzn/base16.git:latest",
```

Comment:
You can't use `latest` in the main plugins repo, you have to use a commit hash.

Reply:
Done